src/test/java/org/kohsuke/github/junit/GitHubWireMockRule.java

```diff
@@ -45,6 +49,12 @@ public class GitHubWireMockRule extends WireMockMultiServerRule {
     private final static boolean useProxy = takeSnapshot
             || System.getProperty("test.github.useProxy", "false") != "false";
 
+    private final static Pattern ACTIONS_USER_CONTENT_PATTERN = Pattern
+            .compile("https://pipelines[a-z0-9]*\\.actions\\.githubusercontent\\.com", Pattern.CASE_INSENSITIVE);
+    private final static Pattern BLOB_CORE_WINDOWS_PATTERN = Pattern
+            .compile("https://([a-z0-9]*\\.blob\\.core\\.windows\\.net)", Pattern.CASE_INSENSITIVE);
+    private final static String ORIGINAL_HOST = "originalHost";
+
```

Comment:
Sorry this is such a mess.  Well done updating it.

Reply:
Yeah, I struggled a lot with it. Note that I don't think it's a limitation of what you did, I was surprised I didn't have some sort of context attached to requests by Wiremock so that I could add the host transparently there.
If at some point, they add a signature to check the URL is not tainted, we will be screwed :/